src/types/siteLaunch.ts

```diff
@@ -47,9 +53,27 @@ export interface SiteLaunchDto {
 export const SITE_LAUNCH_TASKS_LENGTH = (Object.keys(SITE_LAUNCH_TASKS).length -
   1) as SiteLaunchTaskTypeIndex // we minus one here since step 0 is not counted as a step done
 
+export const NEW_DOMAIN_SITE_LAUNCH_TASKS_LENGTH = (Object.keys(
+  NEW_DOMAIN_SITE_LAUNCH_TASKS
+).length - 1) as SiteLaunchTaskTypeIndex
+
 export const SITE_LAUNCH_PAGES = {
   DISCLAIMER: 1,
   INFO_GATHERING: 2,
   RISK_ACCEPTANCE: 3,
   CHECKLIST: 4,
 }
+
+export const TITLE_TEXTS_OLD_DOMAIN = [
+  "Set your DNS Time To Live(TTL) to 5 mins at least 24 hours before launching",
+  "Approve and publish your first review request",
+  "Drop existing domains on Cloudfront",
+  "Delete existing DNS records from your nameserver",
+  "Wait 1 hour to flush existing records",
+] as const
+
+export const TITLE_TEXTS_NEW_DOMAIN = [
+  TITLE_TEXTS_OLD_DOMAIN[
+    NEW_DOMAIN_SITE_LAUNCH_TASKS.APPROVE_FIRST_REVIEW_REQUEST
+  ],
+] as const
```

Comment:
maintability: honestly i'm not a fan of this (using array based `const`) because it's not really descriptive and teh indexes are just raw numbers under the hood. 

if it's not too much effort, would it be possible to change this to const obj

Reply:
sure, 8d5806aad19d3bb374c7982d2ebe358b08b11ed7

I still want to iterate thr the tasks in order, so have introduced util funcs achieve this functionality

Reply:
sure. can just do `_.values` but the `switch` is fine too